backend/filterdefaults/wavedefaults.py

```diff
@@ -1,40 +1,31 @@
 import struct
-from os.path import splitext
 
-from filemanagement import is_path_in_folder
 
-sample_types = {1: "int", 3: "float"}
+def defaults_for_wav_file(file_path):
```

Comment:
This should probably be removed again, since camilladsp now supports wav directly.